packages/components/carousel/src/carousel-item.ts

```diff
@@ -10,3 +10,8 @@ export const carouselItemProps = buildProps({
 } as const)
 
 export type CarouselItemProps = ExtractPropTypes<typeof carouselItemProps>
+
+export type ItemStyle = {
+  transform: string
+  zIndex?: number
+}
```

Comment:
This and other related code seems to be removed.

Reply:
already edited